### PR TITLE
✨ Fast fetch implementation for Valueimpression network

### PR DESCRIPTION
**Pull request by thuyhq**
This commit contains a fast fetch implementation for Valueimpression ad network.
This is my first time participating in the AMP Project, looking forward to helping.

### CODE REVIEW

**Review by powerivq — CHANGES_REQUESTED** [review]

**Comment by thuyhq** [comment]
Hello @powerivq,
Thank you for your review
I have corrected it according to what you asked.

**Comment by powerivq** [comment]
@thuyhq Hi, look like it is not building correctly. Could you check the Travis CI detail and fix whatever issues present? thanks.

**Comment by thuyhq** [comment]
Hi @powerivq,
The errors in Travis CI have been resolved.
What do I need to do next?
Thank you.

**Review by powerivq — APPROVED** [review]

**Comment by powerivq** [comment]
Adding @estherkim for build-system/compile/bundles.config.extensions.json owner approval
Adding @dvoytenko for the addition of the new extension owner approval.

**Review by estherkim — APPROVED** [review]

**Review by dvoytenko — APPROVED** [review]

**Comment by thuyhq** [comment]
Hi @powerivq
When is my pull request merged in the main branch?

**Comment by powerivq** [comment]
@thuyhq could you rebase it on the latest master? Must have all tests passed before merging.

**Comment by thuyhq** [comment]
Hi @powerivq,
I rebase my branch on master. But then circleCI gives error 
ERROR: Error: Build failed with 1 error:
extensions/amp-instagram/1.0/amp-instagram.js:41:20: error: Could not resolve "../../../build/amp-instagram-1.0.css" 

Looks like this error is not coming from my commit. How do I fix them?
Thanks very much.

File: build-system/compile/bundles.config.extensions.json [hunk 1]
```diff
@@ -246,6 +251,7 @@
     "latestVersion": "0.1",
     "options": {"hasCss": true}
   },
+  {"name": "amp-instagram", "version": "1.0", "latestVersion": "0.1"},
```
Comment (powerivq) [inline build-system/compile/bundles.config.extensions.json:254]: See this. You might want to revert this change.

File: build-system/compile/bundles.config.extensions.json [hunk 2]
```diff
@@ -513,6 +519,7 @@
     "latestVersion": "0.1",
     "options": {"hasCss": true}
   },
+  {"name": "amp-stream-gallery", "version": "1.0", "latestVersion": "0.1"},
```
Comment (powerivq) [inline build-system/compile/bundles.config.extensions.json:522]: Same here.

**Review by powerivq — CHANGES_REQUESTED** [review]

**Comment by powerivq** [comment]
There's a check-lint failure https://app.circleci.com/pipelines/github/ampproject/amphtml/3506/workflows/0c17ee5c-0f3c-4497-bc38-0a2ddce0e5c3/jobs/43154

**Review by powerivq — APPROVED** [review]

**Comment by thuyhq** [comment]
Hello @powerivq,
When will my pull request be released?

**Comment by thuyhq** [comment]
Hello @estherkim , @dvoytenko ,
What do I need to do next for my PR to be released?

**Comment by estherkim** [comment]
Hi @thuyhq, looks like it has been in Stable since Mar 10.

**Pull request merged**